node-util/sbin/oo-last-access

```diff
@@ -115,7 +121,8 @@ class LastAccessUpdater
     # Doing the file read with awk is 10x faster than doing it inside
     # Ruby and we desperately need speed here.
     ignore_cond = [@hosts_ignore.map { |a| "$2 != \"#{a}\"" },
-                   @addrs_ignore.map { |a| "$1 != \"#{a}\"" }].flatten.join(' && ')
+                   @addrs_ignore.map { |a| "$1 != \"#{a}\"" },
+                  "$_ !~ /#{ @useragent_ignore.join('|') }/" ].flatten.join(' && ')
```

Comment:
I see some value in allowing regex and defining the option as such in the node.conf.

Why exactly shouldn't we `$_`?